### ISSUE TITLE
non-existent service "contao.image.sizes"

### ISSUE THREAD

**Issue by rustykowski** [heimrichhannot/contao-slick-bundle#76]
**Context**
Contao version: 4.9.42
Bundle version: 1.8.3 and 1.8.4
PHP version: 8.2

**Description**
Update fails with this error-message:
```
vendor/bin/contao-console list
In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:
                                                                               
  The service "HeimrichHannot\SlickBundle\DataContainer\SlickSpreadContainer"  
   has a dependency on a non-existent service "contao.image.sizes".
```
After downgrading to 1.8.2 the error disappeared.

**Comment by koertho** [comment]
Thanks for reporting! Will be fixed when #77 is merged.

**Comment by koertho** [comment]
Fixed in 1.9.0
